src/schoology-extractor/schoology_extractor/api/request_client.py

```diff
@@ -454,3 +454,50 @@ def get_attendance(
         result = self.get(url)
 
         return result["date"]
+
+    def get_discussions(
+        self, section_id: int
+    ) -> list:
+        """
+        Retrieves disucssions list for a section.
+
+        Parameters
+        ----------
+        section_id : int
+            A Section Id
+
+        Returns
+        -------
+        list
+            A list of all parsed results from the server
+        """
+
+        url = f"sections/{section_id}/discussions"
+
+        result = self.get(url)
+
+        return result["discussion"]
```

Comment:
Can you add a note stating that paging is not supported by this endpoint, so that the reader will know that lack of paging support is not an oversight?

Reply:
✔️